docs/static/spec/openapi/logstash-api.yaml

```
@@ -32,52 +32,195 @@ security:
+        '200':
+          description: Indicates a successful call
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/Common'
+                  - $ref: '#/components/schemas/PipelineSettings'
+                  - type: object
+                    properties:
+                      jvm:
+                        type: object
+                        properties:
+                          pid:
+                            type: integer
+                          version:
+                            type: string
+                          vm_name:
+                            type: string
+                          vm_version:
+                            type: string
+                          vm_vendor:
+                            type: string
+                          start_time_in_millis:
+                            type: integer
+                          mem:
+                            type: object
+                            properties:
+                              heap_init_in_bytes:
+                                type: integer
+                              heap_max_in_bytes:
+                                type: integer
+                              non_heap_init_in_bytes:
+                                type: integer
+                              non_heap_max_in_bytes:
+                                type: integer
+                          gc_collectors:
+                            type: array
+                            items:
+                              type: string
+
+  /_node/os:
+    get:
+      summary: Get node-level OS information.
+      description: |
+        Show the OS name, architecture, version, and available processors
+      operationId: nodeInfoOS
+      tags:
+        - node info
+      parameters:
+        - name: pretty
+          in: query
+          schema:
+            type: boolean
+          description: >
+            If you append `?pretty=true` to the request, the JSON returned will be pretty formatted. Use it for debugging only!
+      responses:
+        '200':
+          description: Indicates a successful call
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/Common'
+                  - $ref: '#/components/schemas/PipelineSettings'
+                  - type: object
+                    properties:
+                      os:
+                        type: object
+                        properties:
+                          name:
+                            type: string
+                          arch:
+                            type: string
+                          version:
+                            type: string
+                          available_processors:
+                            type: integer
+
+  /_node/pipelines:
+    get:
+      summary: Get pipeline-specific information
```

Not mandatory, but might make it clearer that this operation potentially retrieves info about multiple pipelines (depending on authority?)
```suggestion
      summary: Get information for pipelines
```